database_docs/doc_base.json

```diff
@@ -1,404 +1,404 @@
+    "energy_recovery_effectiveness": "Energy recovery effectiveness (unitless)",
+    "enthalpy_recovery_ratio": "Enthalpy recovery ratio as per AHRI Standard 1060",
+    "equipment_type": "Type of equipment",
+    "exit_lighting_schedule": "Schedule to use for exit lights",
+    "fan_cooling_application": "Fan cooling application",
+    "fan_type": "Type of cooling tower fan",
+    "fixed_dry_bulb_high_limit_shutoff_temp": "Dry-bulb temperature high limit shutoff for fixed dry-bulb economizer control",
+    "fixed_dry_bulb_is_allowed": "Is fixed dry-bulb temperature economizer control allowed?",
+    "fixed_enthalpy_fixed_dry_bulb_high_limit_shutoff_dry_bulb_temp": "Dry-bulb temperature high limit shutoff for fixed enthalpy and fixed dry-bulb economizer control",
+    "fixed_enthalpy_fixed_dry_bulb_high_limit_shutoff_enthalpy": "Enthalpy high limit shutoff for fixed enthalpy and fixed dry-bulb economizer control",
+    "fixed_enthalpy_fixed_dry_bulb_is_allowed": "Is fixed enthalpy and fixed dry-bulb temperature economizer control allowed?",
+    "fixed_enthalpy_high_limit_shutoff_enthalpy": "Enthalpy high limit shutoff for fixed enthalpy economizer control",
+    "fixed_enthalpy_is_allowed": "Is fixed enthalpy economizer control allowed?",
+    "fluid_type": "Type of fluid",
+    "form": "Type of curve",
+    "frame_width": "Frame width (in.)",
+    "front_side_infrared_hemispherical_emissivity": "Front side infrared hemispherical emissivity",
+    "front_side_solar_reflectance_at_normal_incidence": "Front side solar reflectance at normal incidence",
+    "front_side_visible_reflectance_at_normal_incidence": "Front side visible reflectance at normal incidence",
+    "fuel_type": "Type of fuel used by the device",
+    "gas_type": "Gas used in between panes",
+    "heat_recovery_exempted": "Exempted when condenser heat recovery system is present",
+    "heating_type": "Type of heating element",
+    "high_bay_lighting_schedule": "Schedule to use for high bay lights",
+    "hourly_loss_base": "Hourly loss base coefficient, the hourly loss requirement is calculated as follows: hourly_loss_base + volume * hourly_loss_volume_allowance",
+    "hourly_loss_volume_allowance": "Hourly loss volume derating coefficient, the hourly loss requirement is calculated as follows: hourly_loss_base + volume * hourly_loss_volume_allowance",
+    "hr_1": "Hour 1",
+    "hr_10": "Hour 10",
+    "hr_11": "Hour 11",
+    "hr_12": "Hour 12",
+    "hr_13": "Hour 13",
+    "hr_14": "Hour 14",
+    "hr_15": "Hour 15",
+    "hr_16": "Hour 16",
+    "hr_17": "Hour 17",
+    "hr_18": "Hour 18",
+    "hr_19": "Hour 19",
+    "hr_2": "Hour 2",
+    "hr_20": "Hour 20",
+    "hr_21": "Hour 21",
+    "hr_22": "Hour 22",
+    "hr_23": "Hour 23",
+    "hr_24": "Hour 24",
+    "hr_3": "Hour 3",
+    "hr_4": "Hour 4",
+    "hr_5": "Hour 5",
+    "hr_6": "Hour 6",
+    "hr_7": "Hour 7",
+    "hr_8": "Hour 8",
+    "hr_9": "Hour 9",
+    "independent_variable_1": "Independent variable",
```

Comment:
Do we need to specify independent variable 1 and 2?

Reply:
Thanks! Updated.